packages/generator/src/complex-type/imports.spec.ts

```diff
@@ -94,4 +95,49 @@ describe('complex type imports', () => {
       }
     ]);
   });
+
+  describe('ESM support', () => {
+    const commonjsOptions = {
+      generateESM: false
+    } as CreateFileOptions;
+
+    const esmOptions = {
+      generateESM: true
+    } as CreateFileOptions;
+
+    it('importDeclarations with CommonJS', () => {
```

Comment:
[req] Please change this to read like a sentence. (I know some others in the file read incorrect)
Please include only 2 tests: default behavior (generateESM is false) and when the option is explicitly set to true.